.jscsrc

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
This doesn't look to be up to date. It's missing a `"disallowSpaceAfterKeywords"` list